### ISSUE TITLE
Stop   "auto: true" if only one slide

### ISSUE THREAD

**Issue by mactedder** [stevenwanderski/bxslider-4#607]
I have seen numerous comments about this issue, but is there an official fix for this yet? I am do not write Jquery so would need it in plain english.

Basically I need my slideshow to not 'auto' animate if there is only one slide. Currently my slideshow has a fade animation which looks odd when there is only one item in the slideshow. 

I am surprised this isn't accounted for in the default code?

**Comment by Tidal-Wave** [comment]
Related to https://github.com/stevenwanderski/bxslider-4/issues/712, closing due to duplication of issue and housekeeping. Will address in next update. 

As a note: the code presented here doesn't work for 2 slides because you are testing for `length > 2` instead of `length >= 2`.
